### PR TITLE
chore: Update NuGet package versions

### PR DESCRIPTION
**Pull request by BjarkeMeier**
## References

* #0000

### CODE REVIEW

**Review by johevemi — DISMISSED** [review]

File: source/dotnet/wholesale-api/DatabricksClient/DatabricksClient.csproj
Comment (energinet-ajw) [inline source/dotnet/wholesale-api/DatabricksClient/DatabricksClient.csproj:1]: Are we using Newtonsoft.Json? Or the other one?

File: source/dotnet/wholesale-api/Infrastructure/Calculations/DatabricksCalculatorJobSelector.cs
Comment (energinet-ajw) [inline source/dotnet/wholesale-api/Infrastructure/Calculations/DatabricksCalculatorJobSelector.cs:1]: Do you have a better name than jobs.Jobs?

File: source/dotnet/wholesale-api/DatabricksClient/DatabricksClient.csproj [hunk 1]
```diff
@@ -22,7 +22,8 @@ limitations under the License.
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Databricks.Client" Version="1.1.2515.1" />
+    <PackageReference Include="Microsoft.Azure.Databricks.Client" Version="2.0.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```
Comment (BjarkeMeier) [inline source/dotnet/wholesale-api/DatabricksClient/DatabricksClient.csproj:26]:
```suggestion
```

File: source/dotnet/wholesale-api/DatabricksClient/DatabricksClient.csproj
Comment (BjarkeMeier) [reply source/dotnet/wholesale-api/DatabricksClient/DatabricksClient.csproj:1]: Normally "the other one". But there might be special case exceptions.

**Pull request closed without merging**